src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportCRSSectorStandardToPL.sql

```diff
@@ -0,0 +1,134 @@
+﻿
+CREATE PROCEDURE [dbo].[ImportCRSSectorStandardToPL]
+(
+   @RunId int
+)
+AS
+-- ==========================================================================================================
+-- Author:      Sarma Evani
+-- Create Date: 17/Dec/2020
+-- Description: Import, Transform and Load CRS Framework Presentation Layer Table
+-- ==========================================================================================================
+BEGIN TRY
+		DECLARE @LogID int
+
+		/* Start Logging Execution */
+
+			  INSERT INTO Mgmt.Log_Execution_Results
+				  (
+					RunId
+				   ,StepNo
+				   ,StoredProcedureName
+				   ,StartDateTime
+				   ,Execution_Status
+				  )
+			  SELECT 
+					@RunId
+				   ,'Step-6'
+				   ,'ImportCRSSectorStandardToPL'
+				   ,getdate()
+				   ,0
+
+			  SELECT @LogID=MAX(LogId) FROM Mgmt.Log_Execution_Results
+			  WHERE StoredProcedureName='ImportCRSSectorStandardToPL'
+			  AND RunId=@RunID
+
+			BEGIN TRANSACTION
+
+					DELETE FROM [ASData_PL].[CRS_SectorStandard]
+
+					INSERT [ASData_PL].[CRS_SectorStandard]
+					(												
+						[Id],
+						[Title],
```

Comment:
Can we name it StandardSector Please and Is it Possible to add a Reference Key similar to Framework?

Reply:
Changes are deployed to AT